lib/compat/wordpress-6.5/block-bindings/sources/pattern.php

```diff
@@ -28,7 +28,7 @@
 		}
 	};
 	wp_block_bindings_register_source(
-		'pattern_attributes',
+		'core/pattern-attributes',
```

Comment:
@talldan and @kevin940726, are you settled on the final name for the block binding source for Pattern Overrides? Tomorrow is the last day to change it 😄

Reply:
I'd vote for `core/pattern-overrides` but I wouldn't mind any other choices as long as it's consistent 😄.

Reply:
I agree I would try to keep it as consistent as possible. What is the name of the feature? [Synced pattern overrides](https://github.com/WordPress/gutenberg/issues/53705)?

Reply:
I'm merging this pull request, but let's decide the name and we can change that in another pull request

Reply:
I'd also vote for core/pattern-overrides

Reply:
I've created this pull request to update that: [link](https://github.com/WordPress/gutenberg/pull/58434).